FIP0036/post_process/main.py

```diff
@@ -0,0 +1,127 @@
+"""
+
+from groups import groups
+from tqdm import tqdm
+from preprocess import dataPreprocess
+from counting import countVote
+import pandas as pd
+
+
+
+def run(export_path:str):
+    
+    '''
+    runs the post-processing code. results get exported to export_path
+    
+    Parameters
+    ----------
+    export_path : str
+        where to export the results to
+    
+    Returns
+    -------
+    None, but stores csvs with raw votes, processed votes and tallied votes
+    in export_path
+    
+    
+    '''
+    #
+    HEIGHT = 2162760
+    #####################################################################################
+    #
+    #               gets relevant datasets
+    #
+    ####################################################################################
+    # gets relevant datasets
+    datasets=dataPreprocess(height=HEIGHT,secretString='SecretString.txt')  
+    list_of_votes=datasets['votes']
+    N_votes=len(list_of_votes)
+    signatures=[]
+    deal=groups(1)
+    capacity=groups(2)
+    client=groups(3)
+    token=groups(4)
+    core=groups(5)
+    groups_of_voters={
+        'core':core,
+        'token':token,
+        'client':client,
+        'capacity':capacity,
+        'deal':deal
+        }
+    print('')
+    print('begin counting...')
+    print('')
+    #####################################################################################
+    #
+    #               main loop
+    #
+    ####################################################################################
+    
+    
+    for ii in tqdm(range(N_votes)):
+        vote = list_of_votes.iloc[ii]
+        groups_of_voters,signatures=countVote(vote,groups_of_voters,datasets,signatures)
+    GROUPS=['deal','capacity','client','token','core']
+    
+    
+    #####################################################################################
+    #
+    #               displays tallied votes
+    #
+    ####################################################################################
+    for gr in GROUPS:
+        print('###################')
+        print('Counting '+str(gr))
+        print('###################')
+        print('')
+        print('')
+        try:
+            groups_of_voters[gr].count()
+        except:
+            pass
+        
+        print('')
+    
+    #####################################################################################
+    #
+    #              exports
+    #
+    ####################################################################################    
+    all_votes=[]
+    tallied=[]
+    
+    
+    for gr in groups_of_voters:
+        all_votes.append(groups_of_voters[gr].listVotes)
+            
+        try:
+            tt=groups_of_voters[gr].tally
+            tt['id']=groups_of_voters[gr].groupName
+        except:
+            tt={'Approve': 0,
+             'Reject':0,
+             'id': groups_of_voters[gr].groupName}
+            
+            
+        tallied.append(tallied)
```

Comment:
I am pretty sure that this is just appending the empty initialized list (tallied) to itself (this is why when you cast it as a dataframe the tallied.csv file just has a bunch of empty lists) 

I think what you mean is tallied.append(tt)